DTML.EduBot/Dialogs/LessonDialog.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using AdaptiveCards;
+using DTML.EduBot.LessonPlan;
+using Microsoft.Bot.Connector;
+
+namespace DTML.EduBot.Dialogs
+{
+    using System.Threading.Tasks;
+    using Microsoft.Bot.Builder.Dialogs;
+
+    [Serializable]
+    public class LessonDialog : IDialog<string>
+    {
+        private Lesson lesson;
+        private int currentTopic;
+
+        public LessonDialog(Lesson lesson)
+        {
+            this.lesson = lesson;
+            this.currentTopic = 0;
+        }
+
+        public async Task StartAsync(IDialogContext context)
+        {
+            await PostAdaptiveCard(context);
+            context.Wait(this.CheckAnswerAsync);
+        }
+
+        private async Task PostAdaptiveCard(IDialogContext context)
+        {
+            AdaptiveCard adaptiveCard = new AdaptiveCard()
+            {
+                Body = new List<CardElement>()
+                {
+                    new TextBlock()
+                    {
+                        Text = "This is ________"
```

Comment:
Can you move out this hard coded stuff and store it as data in this.lesson, so the text would look like -> 

Text = lesson.topic.questions[i].text (example)

Let's work with Soumya to build a lesson plan object for her lesson plans.